### PR TITLE
*: back quote and escape MySQL identifier

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
duplicate of #290

### What is changed and how it works?
back quote and escape MySQL identifier (mainly `task.Name` to construct table name)

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

 - Has exported variable/fields change

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: loader/checkpoint.go [hunk 1]
```diff
@@ -79,7 +80,7 @@ type RemoteCheckPoint struct {
 	conn           *DBConn
 	id             string
 	schema         string
-	table          string
+	tableName      string
```
Comment (WangXiangUSTC) [inline loader/checkpoint.go:83]: how about adding a comment, tableName contains schema name

**Comment by lance6716** [comment]
IT is useful 😂 PTAL @WangXiangUSTC @csuzhangxc

**Comment by WangXiangUSTC** [comment]
LGTM

**Pull request merged**